deployer/scripts/hawkular.sh

```diff
@@ -219,37 +179,73 @@ EOF
       )"
    fi
   fi
- 
+
   if [ "${use_persistent_storage}" = true ]; then
     if [ "${dynamically_provision_storage}" = true ]; then
       echo "Setting up Cassandra with Dynamically Provisioned Storage"
       # Deploy the main 'master' Cassandra node
       # Note that this may return an error code if the pvc already exists, this is to be expected and why we have the || true here
-      oc process hawkular-cassandra-node-dynamic-pv -v IMAGE_PREFIX=$image_prefix -v IMAGE_VERSION=$image_version -v NODE=1 -v PV_SIZE=$cassandra_pv_size -v MASTER=true | oc create -f - || true
+      oc process hawkular-cassandra-node-dynamic-pv \
+        -v IMAGE_PREFIX=$image_prefix \
+        -v IMAGE_VERSION=$image_version \
+        -v NODE=1 \
+        -v PV_SIZE=$cassandra_pv_size \
+        -v MASTER=true \
+        -v TRUSTSTORE_CLIENT_AUTHORITIES=${TRUSTSTORE_CLIENT_AUTHORITIES} \
+        -v TRUSTSTORE_NODES_AUTHORITIES=${TRUSTSTORE_NODES_AUTHORITIES} | oc create -f - || true
       # Deploy any subsequent Cassandra nodes
       for i in $(seq 2 $cassandra_nodes);
       do
         # Note that this may return an error code if the pvc already exists, this is to be expected and why we have the || true here
-        oc process hawkular-cassandra-node-dynamic-pv -v IMAGE_PREFIX=$image_prefix -v IMAGE_VERSION=$image_version -v PV_SIZE=$cassandra_pv_size -v NODE=$i | oc create -f - || true
+        oc process hawkular-cassandra-node-dynamic-pv \
+          -v IMAGE_PREFIX=$image_prefix \
+          -v IMAGE_VERSION=$image_version \
+          -v PV_SIZE=$cassandra_pv_size -v NODE=$i \
+          -v TRUSTSTORE_CLIENT_AUTHORITIES=${TRUSTSTORE_CLIENT_AUTHORITIES} \
+          -v TRUSTSTORE_NODES_AUTHORITIES=${TRUSTSTORE_NODES_AUTHORITIES} | oc create -f - || true
```

Comment:
These are there because the default value is `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`, so, we are overriding it here. Should I then just change the default to be like `/hawkular-cassandra-certs/tls.peer.truststore.crt` ?